datafusion/physical-expr/src/datetime_expressions.rs

```diff
@@ -783,19 +788,11 @@ mod tests {
         ]);
         assert!(res.is_ok());
 
-        //
-        // Fallible test cases
-        //
```

Comment:
This is not only for "invalid number of arguments". All test cases under this section are fallible. We should keep it.